@stellar/lyra-api/package.json

```diff
@@ -1,31 +1,21 @@
 {
   "name": "@stellar/lyra-api",
-  "main": "build/index.min.js",
-  "types": "build/@stellar/lyra-api/src/index.d.ts",
-  "prettier": "@stellar/prettier-config",
   "version": "1.0.0-alpha.2",
+  "license": "Apache-2.0",
+  "author": "Stellar Development Foundation <hello@stellar.org>",
   "description": "Utility functions to interact with Lyra extension",
   "repository": {
     "url": "git@github.com:stellar/lyra.git",
     "type": "git"
   },
-  "author": "Stellar Development Foundation <hello@stellar.org>",
-  "browserslist": {
-    "production": [
-      "last 3 chrome version",
-      "last 3 firefox version"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version"
-    ]
-  },
-  "license": "MIT",
-  "lint-staged": {
-    "src/**/*.ts?(x)": [
-      "eslint --fix --max-warnings 0"
-    ]
+  "main": "build/index.min.js",
+  "scripts": {
+    "prepare": "yarn build",
+    "build": "webpack --config webpack.config.js --mode production",
+    "start": "webpack --config webpack.config.js --watch --mode development",
+    "test": "jest `pwd`/src"
```

Comment:
This is set up such that there's only 1 jest "instance" for all workspaces, so by default `jest` will run all tests in all projects. This line scopes the test run to only tests under the specific workspace's directory.